### DISCUSSION TITLE
Do we need to change the project name?

### DISCUSSION THREAD

**Discussion by stevenlu137** [myshell-ai/AIlice#52] [Q&A]
It seems that to native English speakers, "AIlice" might come across as an unusual name (it might be more easily interpreted as AI+lice rather than AI+Alice). Should we consider changing the project name? What name would be a good alternative?

I would greatly appreciate everyone's suggestions.

**Comment by phildougherty** [comment]
Yes, it does read a bit as if it is something that is going to colonize your scalp!

Just thinking of other female names with AI in them...Daisy?
Male names: Aiden

**Reply by stevenlu137** [reply]
Even in my culture, Daisy is a beautiful name. Aiden is also good, but it seems that some AI projects have already adopted this name. 
Let's note down these two options for now and decide after we've gathered enough ideas.

**Reply by stevenlu137** [reply]
It looks like Daisy is a great choice; it can even be considered an acronym for "Dynamic AI agent SYstem" in a certain sense :-)

**Reply by KingLeoJr** [reply]
A few suggestions if you change your mind from 'Daisy': Axon, Synapse, Prism, Matrix.

**Reply by stevenlu137** [reply]
I’ve also considered the name "Synapse"! Thank you for offering new options! We’re still open to choices for the name, and I’ll finalize it when I have some time.
